src/components/interaction-designer/block-types/Core_SetGroupMembershipBlock.vue

```diff
@@ -0,0 +1,125 @@
+
+      <div class="form-group">
+        <label>{{'flow-builder.action-label' | trans}}</label>
+        <vue-multiselect v-model="selectedAction"
+                         track-by="id"
+                         label="name"
+                         :placeholder="'flow-builder.action-placeholder' | trans"
+                         :options="actionsList"
+                         :allow-empty="true"
+                         :show-labels="false"
+                         :searchable="false">
+        </vue-multiselect>
+      </div>
+
+      <group-selector :block="block"/>
+
+      <first-block-editor-button
+        :flow="flow"
+        :block-id="block.uuid"/>
+
+    </fieldset>
+
+    <block-id :block="block"/>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { namespace } from 'vuex-class'
+import { Component, Prop } from 'vue-property-decorator'
+
+import {
+  IFlow,
+  IBlock,
+  ISetGroupMembershipBlockConfig,
+} from '@floip/flow-runner'
+import BlockNameEditor from '../block-editors/NameEditor.vue'
+import BlockLabelEditor from '../block-editors/LabelEditor.vue'
+import BlockSemanticLabelEditor from '../block-editors/SemanticLabelEditor.vue'
+import FirstBlockEditorButton from '../flow-editors/FirstBlockEditorButton.vue'
+import BlockId from '../block-editors/BlockId.vue'
+import GroupSelector from '@/components/interaction-designer/block-editors/GroupSelector.vue'
+import VueMultiselect from 'vue-multiselect'
+
+import SetGroupMembershipStore, { BLOCK_TYPE, ADD_KEY, REMOVE_KEY } from '@/store/flow/block-types/Core_SetGroupMembershipStore'
+import lang, { trans } from '@/lib/filters/lang'
+import { createDefaultBlockTypeInstallerFor } from '@/store/builder'
+import { find } from 'lodash'
+
+const blockVuexNamespace = namespace(`flow/${BLOCK_TYPE}`)
+const flowVuexNamespace = namespace('flow')
+const builderVuexNamespace = namespace('builder')
+
+interface IGroupOption {
+  id: string;
+  name: string;
+}
+
+//providing this generic is required by tsserver checking but not in the build run by yarn storybook
+//TODO - understand what is going on here and if there is something more correct we should have instead
+@Component<any>({
+  components: {
+    BlockNameEditor,
+    BlockLabelEditor,
+    BlockSemanticLabelEditor,
+    FirstBlockEditorButton,
+    BlockId,
+    GroupSelector,
+    VueMultiselect,
+  },
+  mixins: [lang],
+})
+class Core_SetGroupMembershipBlock extends Vue {
+  @Prop() readonly block!: IBlock
+  @Prop() readonly flow!: IFlow
+
+  actionsList: IGroupOption[] = [
+    {
+      id: ADD_KEY,
+      name: trans('flow-builder.add'),
+    },
+    {
+      id: REMOVE_KEY,
+      name: trans('flow-builder.remove'),
+    },
+  ]
+
+  get selectedAction() {
+    const { isMember } = this.block.config as ISetGroupMembershipBlockConfig
+    //TODO: we can remove the safe cast JSON.parse(isMember) once ISetGroupMembershipBlockConfig isMember type is changed to boolean
```

Comment:
> [...] once ISetGroupMembershipBlockConfig isMember type is changed to boolean

When does this happen?

Reply:
@bzabos , honestly, I don't know who should I contact for flow-runner change request like this. I just notified you with Roan & Bulat on slack https://votomobile.slack.com/archives/CMQDVRDN3/p1616178709010700 
I can create a ticket for it, but I'm not sure I have the full context in case this is already handled in a specific ticket, so I need your opinion here.